src/test/java/org/arquillian/container/chameleon/ConfigurationTestCase.java

```diff
@@ -80,4 +80,22 @@ public void shouldFailOnMissingContainerFile() throws Exception {
         configuration.setChameleonTarget("wildfly:8.2.0.Final:managed");
         configuration.validate();
     }
+
+    @Test
+    public void shouldReturnFalseForMissingContainerName() throws Exception {
+        boolean isSupported = ChameleonConfiguration.isSupported("MISSING_TARGET:8.2.0.Final:managed");
+        Assert.assertFalse(isSupported);
+    }
+
+    @Test
+    public void shouldReturnTrueForGivenContainerName() throws Exception {
+        boolean isSupported = ChameleonConfiguration.isSupported("wildfly:8.2.0.Final:managed");
```

Comment:
we have coordinate information like follow:
```json
 {
        "artifact_id": "jboss-as-arquillian-container-remote",
        "group_id": "org.jboss.as",
        "name": "Arquillian Container JBoss AS Remote 7.x",
        "container_type": "REMOTE",
        "configurations": [
            {
                "type": "java.lang.String",
                "description": "The ip address of the running server instances management interface",
                "name": "managementAddress",
                "default": "127.0.0.1"
            },
            {
                "type": "java.lang.Integer",
                "description": "The management port for deployment",
                "name": "managementPort",
                "default": "9999"
            }
        ]
    },
```

So from this information we couldn't check that this is supported by chameleon or not? 
So we have to check either it is in chameleon.yml in chameleon & parse it OR check using method.